### ISSUE TITLE
Significant performance improvement on rollup build with minify "sequence=false"

### ISSUE THREAD

**Issue by sod** [jspm/jspm-cli#1946]
Our build currently sits at unminified 2,5 MB. Compress/Mangle took 1 minute with uglify 2.6.2, takes 40s with uglify 2.6.4 and drops to 10s in uglify 2.6.4 if the compressor option `sequence=false` is set.

Would be nice to be able to set  `sequence=false` from the commandline or maybe just set it by default for rollup builds. The change in filesize is insignificant.

```sh
$ time uglifyjs --mangle -c sequences=false,warnings=false -- build.js > build.no-sequences.min.js

user	0m9.916s
```

```sh
$ time uglifyjs --mangle -c sequences=true,warnings=false -- build.js > build.with-sequences.min.js

user	1m47.614s
```

```sh
$ uglifyjs --version
uglify-js 2.6.4
```

```sh
$ ll build.*
-rw-r--r--  1 alex  staff  1817452 26 Jun 12:24 build.js
-rw-r--r--  1 alex  staff   897958 26 Jun 12:26 build.no-sequences.min.js
-rw-r--r--  1 alex  staff   895331 26 Jun 12:28 build.with-sequences.min.js
```

The test build file: https://gist.githubusercontent.com/sod/0918ba26c829f6c0800cde2213c40fad/raw/a56131b17e8d3599e00acdad54535eb9faa71fcb/build.txt

**Comment by sod** [comment]
The folks at uglifyjs replied incredibly quickly. If https://github.com/mishoo/UglifyJS2/pull/1172 lands with a sequences default of 20, nothings needs to be done here :)

**Comment by guybedford** [comment]
We could just set the sequences option to 20 by default internally. That would be done in https://github.com/jspm/jspm-cli/blob/0.17/lib/bundle.js#L69 via `opts.uglify = { compressor: { sequences: 20 } }` I believe? (and in https://github.com/jspm/jspm-cli/blob/0.17/lib/bundle.js#L105 as well).

A PR would be welcome if you'd like to test that out?

**Comment by sod** [comment]
Sure, I can do a PR if uglify merges the sequence change with anything other as 20.

**Comment by guybedford** [comment]
What I mean is we can implement something now by setting the option as a jspm default without having to wait for the Uglify release.

**Comment by sod** [comment]
The `sequence` option currently only accepts boolean (true = limit to 2000, false = disable sequence optimization). So it is not possible to set it to `20` right now. And `false` would negatively impact the output filesize.

**Comment by guybedford** [comment]
Ah right, I misunderstood that. Sure we can wait for that PR to be released then.

**Comment by sod** [comment]
sequence PR got merged and released in [uglify-js@2.7.0](https://github.com/mishoo/UglifyJS2/releases/tag/v2.7.0) - is automatically installed with jspm@next as the version in package.json `^2.6.1` installs 2.7.0. 

30% reduces bundle + minify time for me :)

**Comment by guybedford** [comment]
Excellent, thanks for the follow up!
